include/libtorrent/session_handle.hpp

```diff
@@ -456,6 +456,14 @@ namespace libtorrent {
 		// posted, regardless of the alert mask.
 		void dht_live_nodes(sha1_hash const& nid);
 
+		// Query the DHT to retrieve a sample of the infohashes that the specific
+		// node currently have in their storage.
```

Comment:
actually, this call will only send a single request to a single node, right? Perhaps the wording here can be clearer to say "Query the DHT node specified by ``ep``". This isn't querying the DHT as a whole, right? Since there's no traversal or any use of our routing table. (assuming I understand this correctly)